tests/DMSDocumentTest.php

```diff
@@ -65,12 +67,24 @@ public function testDocumentHasCmsFieldForManagingRelatedDocuments()
     }
 
     /**
+     * Ensures that the DMS Document CMS Related and Versions fields are removed if user can't edit
+     */
+    public function testDocumentHasNoCMSFieldsForManagingRelatedDocumentsIfCantEdit()
+    {
+        $this->logInWithPermission('another-user');
+        $document = $this->objFromFixture('DMSDocument', 'doc-only-these-users');
+        $gridField = $this->getGridFieldFromDocument($document);
+        $this->assertNull($gridField);
```

Comment:
Maybe just rename `getGridFieldFromDocument` to `getRelatedDocumentsGridField` so it's more clear?